assign_rights/tests.py

```diff
@@ -119,6 +119,21 @@ def test_calculate_dates(self):
         for granted in shell.rightsgranted_set.all():
             self.check_object_dates(granted, request_start_date, request_end_date)
 
+    def test_create_json(self):
+        shell = random.choice(RightsShell.objects.all())
+        request_end_date = random_date().isoformat()
+        request_start_date = random_date().isoformat()
+        start_date, end_date = self.assembler.calculate_dates(shell, request_start_date, request_end_date)
+        serialized = self.assembler.create_json(shell, start_date, end_date)
+
+        granted_list = []
+        for granted in shell.rightsgranted_set.all():
+            start_date, end_date = self.assembler.calculate_dates(granted, request_start_date, request_end_date)
+            granted_list.append(self.assembler.create_json(granted, start_date, end_date))
+        for item in granted_list:
+            serialized["rights_granted"].append(item)
+        self.assertTrue(isinstance(serialized, dict))
```

Comment:
Would definitely validate here!

Reply:
Do you mean validate against the RAC json schema?